13/umbraco-engage/upgrading/upgrade-umbraco-engage.md

```diff
@@ -0,0 +1,49 @@
+# Upgrade Umbraco Engage
+
+This article shows how to upgrade Umbraco Engage to the latest version.
+
+When upgrading Umbraco Engage, be sure to consult the [version specific upgrade](version-specific-upgrade-notes.md) notes to learn about potential breaking changes and common pitfalls.
+
+{% hint style="warning" %}
+Before upgrading, it is recommended to backup your site and database.
+{% endhint %}
+
+## Get the latest version of Umbraco Engage
+
+To upgrade to the latest version of Umbraco Engage you can use:
+
+* NuGet
+* Visual Studio
+
+### NuGet
+
```

Comment:
Umm since it's an upgrade article should the below points mention upgrade instead of install? Or maybe remove the term install and keep it more generic to not confuse if it's install or upgrade steps.

Reply:
So, the way you upgrade these add on packages, is that you install the latest version on top of the existing one.
Therefor it is actually correct that you need to install the package 💪

Reply:
That also makes sense 😅